utilities/utilities/testing/todoist.py

```diff
@@ -0,0 +1,122 @@
+from datetime import datetime
+from typing import Dict, List, Optional
+from uuid import uuid4
+
+from todoist_api_python.models import Project, Section, Task
+
+from utilities.api.todoist_api import TodoistHelper
+
+DEFAULT_SECTION = "section"
+
+
+class LocalTodoistConnection:
+    tasks: List[Task] = []
```

Comment:
nit: do we want cleaner like init later so not at the start immutable?

Reply:
a bit nicer, will implement